src/transformers/modeling_albert.py

```diff
@@ -757,10 +757,9 @@ def forward(
         position_ids=None,
         head_mask=None,
         inputs_embeds=None,
-        masked_lm_labels=None,
```

Comment:
I think `masked_lm_labels` was deleted accidently here

Reply:
Renamed to `labels`